src/main/java/com/jagrosh/giveawaybot/commands/CreateCommand.java

```diff
@@ -54,17 +54,17 @@ public CreateCommand(Bot bot, EventWaiter waiter) {
     }
     
     @Override
-    protected void execute(CommandEvent event) 
+    protected void execute(CommandEvent event)
     {
         List<Giveaway> list = bot.getDatabase().giveaways.getGiveaways(event.getGuild());
         if(list==null)
         {
-            event.replyError("An error occurred when trying to being giveaway creation.");
+            event.replyError("An error occurred when trying to create the giveaway.");
```

Comment:
This fixed the typo, but it should have been `begin giveaway creation` to differentiate it from the other error messages.